redis/src/connection.rs

```diff
@@ -1027,10 +1095,18 @@ impl<'a> Drop for PubSub<'a> {
 /// connection.  It only contains actual message data.
 impl Msg {
     /// Tries to convert provided [`Value`] into [`Msg`].
+    #[allow(clippy::unnecessary_to_owned)]
```

Comment:
why is this necessary?

Reply:
It's a false positive recommended change won't build or work.